includes/class-sensei-grading.php

```diff
@@ -1131,11 +1131,34 @@ public static function get_graded_lessons_sum() {
 
 	}
 
+	/**
+	 * Get average grade of all lessons graded
+	 *
+	 * @since 4.2.0
+	 * @return double $graded_lesson_avg
+	 */
+	public static function get_graded_lessons_avg() {
+		$grade_args        = array(
+			'type'     => 'sensei_lesson_status',
+			'status'   => 'any',
+			'meta_key' => 'grade',  // phpcs:ignore Slow query ok.
+		);
+		$grade_count       = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_percentage', $grade_args ), false );
```

Comment:
Ditto for this filter.

Reply:
[Here](https://github.com/Automattic/sensei/pull/4865/files#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR246)